charts/galley/values.yaml

```diff
@@ -95,6 +95,15 @@ config:
             verificationExpiration: 86400
             acmeDiscoveryUrl: null
           lockStatus: unlocked
+      mlsMigration:
+        defaults:
+          status: disabled
+          config:
+            # startTime: "2029-05-16T10:11:12.123Z"
+            # finaliseRegardlessAfter: "2029-10-17T00:00:00.000Z"
+            usersThreshold: 0
+            # clientsThreshold: 50
+          lockStatus: locked
```

Comment:
Not sure what you mean by "safe" defaults. `lockStatus` is better set to `locked` as default, because that's the only configuration we have discussed, and team settings won't implement changing migration settings anyway. The rest is fine, but having `null` vs having a missing field doesn't make a difference.

Reply:
I've now applied this change without the "locked" part.